assets/scss/section.scss

```diff
@@ -1,5 +1,36 @@
+/*doc
+---
+title: Section
+---
+Used to group sets of related components
+into discrete sections by adding padding
+around it's children.
+*/
+
+$sectionSpaceMultiple: 2;
+
 @include responsiveBase() {
 	.section {
-		padding: $base*2 $base 0;
+		padding: $base*$sectionSpaceMultiple $base 0;
+	}
+
+	.section.bordered{
+		padding-bottom: $base*$sectionSpaceMultiple - $base;
+		&:before{
+			left: $base;
+			right: $base;
+		}
+	}
+}
+
+.section.bordered {
```

Comment:
might also be more future-forward to nest the two class selectors here as well, your call